### ISSUE TITLE
CMake export targets properly

### ISSUE THREAD

**Issue by rxdu** [versatica/libmediasoupclient#169]
Hi, the current CMake setup doesn't export the targets properly. It doesn't generate the mediasoupclientTargets.cmake and an external projects won't be able to find the libmediasoupclient project targets from outside easily. I was wondering if this kind of changes will be of interest to you. If yes, I would be glad to submit a pull request.

**Comment by ibc** [comment]
Yes, PR is welcome. It's just that we are terribly busy these months and cannot review/merge as quickly as we'd like.

**Comment by jmillan** [comment]
Closing. Any collaboration is welcome.
